wls-gui-wahllokalsystem/src/stores/stimmabgabevermerkeStore.ts

```diff
@@ -1,13 +1,120 @@
 import type { Stimmabgabevermerke } from "@/types/stimmabgabevermerke/Stimmabgabevermerke.ts";
+import type { Vermerke } from "@/types/stimmabgabevermerke/Vermerke.ts";
+import type { Wahldaten } from "@/types/stimmabgabevermerke/Wahldaten.ts";
 
 import { defineStore } from "pinia";
-import { ref } from "vue";
+import { computed, ref } from "vue";
+
+import { StimmzettelStimmzettelartEnum } from "@/types/stimmabgabevermerke/StimmzettelStimmzettelartEnum.ts";
 
 export const useStimmabgabevermerkeStore = defineStore(
   "stimmabgabevermerke",
   () => {
     const stimmabgabevermerke = ref<Stimmabgabevermerke | null>(null);
 
-    return { stimmabgabevermerke };
+    const lowestNumberOfRowsOverAllWahldaten = computed(() => {
+      const wahldatenIterator = stimmabgabevermerke.value?.wahldaten.values();
+      if (!wahldatenIterator) {
+        return 0;
+      }
+
+      let minVermerkeLength = Infinity;
+      for (const wahldaten of wahldatenIterator) {
+        const currentVermerkeLength = wahldaten.vermerke.length;
+        if (currentVermerkeLength < minVermerkeLength) {
+          minVermerkeLength = currentVermerkeLength;
+        }
+      }
+
+      return minVermerkeLength === Infinity ? 0 : minVermerkeLength;
+    });
+
+    const stimmabgabevermerkeTableTotalEachWahldaten = computed(() => {
+      const totalsForWahldaten: number[] = [];
+
+      stimmabgabevermerke.value?.wahldaten.forEach((wahldaten: Wahldaten) => {
+        const totalVermerke = wahldaten.vermerke.reduce((sum, vermerk) => {
+          return sum + (vermerk.stimmzettel[0]?.anzahl ?? 0);
+        }, 0);
+
+        totalsForWahldaten.push(totalVermerke);
+      });
+
+      return totalsForWahldaten;
+    });
+
+    function _increaseRows(newRowSize: number) {
+      const currentLowestRows = lowestNumberOfRowsOverAllWahldaten.value;
+
+      if (currentLowestRows !== null && currentLowestRows !== undefined) {
```

Comment:
kann man das nicht vereinfachen?
```suggestion
      if (currentLowestRows) {
```

Reply:
nein weil currentLowestRows 0 sein kann und 0 ist ein falsy